continuous_integration/envs/39-dev.yaml

```diff
@@ -15,10 +15,11 @@ dependencies:
   - pytest
   - pytest-cov
   - hilbertcurve
+  - pymorton
   # optional dependencies
   - pyarrow
-  - pip
   - pip:
+    - git+https://github.com/trevorprater/pymorton.git@master
```

Comment:
Keep `pip` in there and install `pymorton` from PyPI like in the other envs. We don't need its master here.

Reply:
resolved in 9756156